en/identity-server/7.0.0/docs/references/extend/user-mgt/provisioning/extend-scim2-user-schemas.md

```diff
@@ -11,7 +11,22 @@ The System for Cross-domain Identity Management (SCIM) is a specification that i
 
 The SCIM 2.0 (System for Cross-Domain Identity Management) specification defines a [fixed set of default attributes](https://tools.ietf.org/html/rfc7643#section-8.2){:target="_blank"} for the user object. This set is defined to ensure the interoperability and it can cater to most of the industry's identity management requirements. However, in reality, organizations have their own attributes defined for their users which have already been used in their LDAP schemas. Therefore, SCIM is extensible enough to cope with custom attributes and uses the [Enterprise User Extension](https://tools.ietf.org/html/rfc7643#section-8.3){:target="_blank"} to support extra attributes for the SCIM user object.
 
-WSO2 Identity Server allows users to define their own user schema in addition to the core user schema. These configured schema are then used while creating or validating user objects. This means that custom user attributes can be passed using SCIM for identity management requirements. Follow the steps given below to add a custom attribute. 
+WSO2 Identity Server allows users to define their own user schema in addition to the core user schema. These configured schema are then used while creating or validating user objects. This means that custom user attributes can be passed using SCIM for identity management requirements.
+
+!!! Note
+    From 6.0.0 onwards, we can use the [SCIM2 Custom User Schema]({{base_path}}/guides/users/attributes/manage-scim2-attribute-mappings) to add custom attributes of the user.
+
+    Reasons why we introduced custom schema to add custom attributes: 
+        1. According to the current model, Enterprise User Extension attributes are in a file, and those configurations are applicable at the server level. 
+        2. Enterprise User Extension is a defined schema, and it should not be allowed to be modified.
+    
+    So it is recommended to use the SCIM2 Custom User Schema to add custom attributes of the user instead of Enterprise User Schema.
```

Comment:
```suggestion
    
```

Reply:
Addressed with https://github.com/wso2/docs-is/pull/4594/commits/1eeb79cb3618689ba3216361d1cd3caddd7b30de